lib/screens/app/app.dart

```diff
@@ -183,21 +183,19 @@ class _AppState extends State<App> with WidgetsBindingObserver {
 
   Widget buildAppBar(BuildContext _context) {
     return AppBar(
-      title: Text(
-        navigationTabs[index].title,
-        style: TextStyle(color: Colors.black87),
-      ),
+      title: Text(navigationTabs[index].title),
       centerTitle: true,
       actions: <Widget>[
         Padding(
-          padding: const EdgeInsets.only(right:17),
+          padding: const EdgeInsets.only(right: 17),
           child: IconButton(
-            icon: Icon(Icons.qr_code_scanner, color: Colors.black, size: 33,),   
-            onPressed: () => NavigationService.of(context).navigateTo(Routes.scanQRCode)),
+              icon: Icon(
+                Icons.qr_code_scanner,
```

Comment:
This seems reverted to what it was

Is the QR scanner icon visible now? Better to define a color since the last time it suddenly vanished when it was replaced with a white icon...

Reply:
As a fix I made the icon 33 px (since the new icon was very small)... and set the bg color to black.

Intentional change back here?